conda_build/build.py

```diff
@@ -582,7 +582,7 @@ def test(m, verbose=True, channel_urls=(), override_channels=False):
         else:
             test_file = join(tmp_dir, 'run_test.sh')
             # TODO: Run the test/commands here instead of in run_test.py
-            cmd = ['/bin/bash', '-x', '-e', test_file]
+            cmd = ['/bin/sh', '-x', '-e', test_file]
```

Comment:
> sh, as it's the only reliable shell that one can find at a fixed location on any POSIX compliant OS.

We've never promised for conda-build to work on a POSIX OS other than Linux (and OS X :-).

> To be honest, let's not over-engineer this change, it's a 0 cost issue

I really can't say because I don't know the internals of conda-build. It probably relies on running our scripts on bash, but I can't tell.

To minimize even further the impact of this PR, what if you change to

```
if sys.platform.startswith('linux'):
    cmd = ['/bin/bash', '-x', '-e', build_file]
else:
    cmd = ['/bin/sh', '-x', '-e', build_file]
```

?